aws/resource_aws_iam_group_membership_test.go

```diff
@@ -232,8 +232,110 @@ resource "aws_iam_group" "group" {
 
 resource "aws_iam_group_membership" "team" {
 	name = "%s"
-	users = ["${aws_iam_user.user.*.name}"]
 	group = "${aws_iam_group.group.name}"
+	users = [
+		"${aws_iam_user.user.*.name[0]}",
```

Comment:
I feel like this /may/ want a TODO?

Reply:
One does not simply forget `"${aws_iam_user.user.*.name[100]}",` 😂 I'll add a comment to both the huge expanded lists post-merge explaining them and hopefully they are caught during 0.12 configuration upgrades.